src/withNetworkConnectivity.js

```diff
@@ -83,7 +83,12 @@ const withNetworkConnectivity = ({
       }
 
       setupConnectivityCheckInterval(() => {
-        if (this.state.isConnected && checkIntervalOfflineOnly) {
+        let isConnected = this.state.isConnected;
```

Comment:
Could you please destructure the store to avoid that long property access?

```js
const { store } = this.context;
if (withRedux && store) {
	store.getState().network.isConnected;
}

...
```

Also, could you also rephrase the comment to "When using redux integration, the connection state is coming from the store"

Thanks!